frontend/compose-html-ext/src/jsMain/kotlin/com/varabyte/kobweb/compose/css/Transform.kt

```diff
@@ -5,6 +5,30 @@ package com.varabyte.kobweb.compose.css
 
 import org.jetbrains.compose.web.css.*
 
+// https://developer.mozilla.org/en-US/docs/Web/CSS/backface-visibility
+class BackfaceVisibility private constructor(private val value: String) : StylePropertyValue {
+
+    override fun toString() = value
+
+    companion object {
+
+        /* Keyword values */
+        val Visible get() = BackfaceVisibility("visible")
+        val Hidden get() = BackfaceVisibility("hidden")
+
+        /* Global values */
+        val Inherit get() = BackfaceVisibility("inherit")
+        val Initial get() = BackfaceVisibility("initial")
+        val Revert get() = BackfaceVisibility("revert")
+        val RevertLayer get() = BackfaceVisibility("revert-layer")
+        val Unset get() = BackfaceVisibility("unset")
+    }
+}
+
+fun StyleScope.backFaceVisibility(backFaceVisibility: BackfaceVisibility) {
+    property("backface-visibility", backFaceVisibility)
```

Comment:
```suggestion
fun StyleScope.backfaceVisibility(backfaceVisibility: BackfaceVisibility) {
    property("backface-visibility", backfaceVisibility)
```
The class name change is good, but the function and parameter names should be updated as well. This goes for the `Modifier` function too.